### PR TITLE
Reports

### PR DESCRIPTION
**Pull request by ericlesaquiles**
Updated README.org and REPORTS.org.
README.org now contains (and is expected to contain, from now on) a list of milestones, with its due details and tags.
REPORTS.org now is updated with the passing tests. The update (as well as the file) was done automatically, by a script, however, so, although I trust it is mostly accurate, I didn't really check all the possible cases, so there might be slight inconsistencies (even taking that apart, there certainly is room for improvement), but it should be more informative and useful for now.

### CODE REVIEW

**Comment by Affonso-Gui** [comment]
are you sure failures is 255 not 2555?

**Comment by ericlesaquiles** [comment]
no, that's a misteak

File: eus-test.l [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env roseus
 (require :unittest "lib/llib/unittest.l")
 
+(load "auxiliary/eus-multiple-values.l")
```
Comment (Affonso-Gui) [inline eus-test.l]: maybe shift this line below, around `;;LOAD AUXILIARY` at #L135?

Comment (ericlesaquiles) [reply eus-test.l]: the thing is that that file might be of use not only for the test files, but for `eus-test.l`as well, so I thought leaving it closer to the header would make that clearer. but I could change it, it is not a problem

Comment (Affonso-Gui) [reply eus-test.l]: for now `eus-test.l` is not using any multiple-value so it should be fine.

Another possibility is to bring `eus-loop.l` here, so we can use both multiple-values and loop in the rest of the file.

**Review by Affonso-Gui — APPROVED** [review]

**Pull request merged**